grizzly-ls/src/grizzly_ls/__main__.py

```diff
@@ -134,7 +134,10 @@ def setup_debugging(args: argparse.Namespace) -> None:
                 logging.info('Waiting for debugger to attach')
                 debugpy.wait_for_client()
         except ModuleNotFoundError:
-            logging.error('Debugging requires the debugpy package to be installed')
+            msg = 'Debugging requires the debugpy package to be installed'
+            logging.error(msg)
```

Comment:
this should be removed. the overloaded `show_message` in `GrizzlyLanguageServer` will call the logger and show the message it's called with. so this message will be logged twice right now.